tests/modelbench_tests/test_record.py

```diff
@@ -78,7 +78,7 @@ def test_v1_hazard_definition_without_tests_loaded():
 
 def test_v1_hazard_definition_with_tests_loaded():
     hazard = SafeHazardV1("dfm", Locale.EN_US, "practice")
-    hazard.tests({"together": {"api_key": "ignored"}, "modellab_files": {"token": "ignored"}})
+    hazard.tests({"together": {"api_key": "ignored"}})
```

Comment:
This will break the test with French prompts.

Reply:
But this hazard is specific to english

Reply:
oh, duh.